app/views/support_interface/references/reinstate.html.erb

```diff
@@ -4,20 +4,19 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      Are you sure you want to reinstate this reference?
+      Are you sure you want to undo the referee's refusal to provide a reference?
     </h1>
 
     <p class="govuk-body">
-      Reinstating a reference will:
+      This will:
     </p>
 
     <ul class="govuk-list govuk-list--bullet">
       <li>allow this referee to provide a reference</li>
-      <li>prevent the candidate from providing a replacement referee</li>
     </ul>
 
     <p class="govuk-body">
-      Reinstating a reference does <strong>not send email notifications</strong>.
+      Undoing a refusal does <strong>not send email notifications</strong>.
```

Comment:
Might be out of scope for this card, but I do wonder whether we should be sending an email here. Mainly just so the referee has something to click on. When I've done this via console I've tended to send a chaser.
 
The other option is that we just tell support colleagues that they need to go back and click on the original link.

I guess there are situations where we might not want to send an email though.

What do you think?

Reply:
Good question. I'd tend towards producing the minimal set of side effects possible from this action. I think this happens infrequently enough that asking support colleagues to tell referees to use the original email is sustainable. Can always manually send the email if for some reason it's asked for especially.